CHANGELOG.md

```diff
@@ -39,6 +39,9 @@
 
 -   Adds for For You tab with styled headers - luc
 -   Creates tab view container for new home - luc
+-   Adds Live and timed auctions in home view - luc
+-   Tab bar text now aligned and centered correctly - luc
+-   Adds iPad support for Auctions view layout - luc
```

Comment:
Ah - these are all on the wrong section also, I'll merge this ( as it's not exactly a PR blocker ) and edit the CHANGELOG on my next PR